app/views/layouts/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Online Marketplace <%= "|#{@page_title}" if @page_title %></title>
+  <%= stylesheet_link_tag 'application', media: 'all' %>
+</head>
+<body>
+
+<div id="header">
+  <h1>Online Marketplace</h1>
```

Comment:
Translations missing

Reply:
Bhai, I have added it from my PR [4-Controllers](https://github.com/nor-wtag/online-marketplace/pull/10). Thank you for the suggestion.